lib/ibazel_watcher.js

```diff
@@ -0,0 +1,27 @@
+const {createInterface} = require('readline');
+
+// ibazel will write this string after a successful build
+// We don't want to re-trigger tests if the compilation fails, so
+// we should only listen for this event.
+const IBAZEL_NOTIFY_BUILD_SUCCESS = 'IBAZEL_BUILD_COMPLETED SUCCESS';
+
+function watch(fileList) {
+  // ibazel communicates with us via stdin
+  const rl = createInterface({input: process.stdin, terminal: false});
+  rl.on('line', chunk => {
+    if (chunk === IBAZEL_NOTIFY_BUILD_SUCCESS) {
```

Comment:
Do you want to notify the user when there is a build failure?

Reply:
Any suggestions how to do that? Pretend there's a test for compilation and that test failed?

Reply:
Yeah, that seems pretty reasonable. After some thinking I can't come up with anything better. If I think of something I'll send a PR.